### ISSUE TITLE
Unknown Sources

### ISSUE THREAD

**Issue by Truckster78** [tgardner/cordova-galaxygear#9]
Hello, 
I use this plugin to connect an Ionic project to a Samsung Gear S2 watch. This, in order to send strings from the watch to the Android smartphone (Galaxy S5 or Galaxy Note3 4G) or in the opposite sens.
I use in my project the JS code, provided as example, in the usage guide page.

The problem is : 
the application is build and installed to the smartphone without problem. But when the  app start up, I get an "error" message which says that (translated from french): 
_CORDOVA_GALAXY_APP does not come from Samsung Galaxy Apps and do not include any app for your Gear. Your device and data are more vulnerable to attacks by unknown sources apps. The installation will be cancelled and the app CORDOVA_GALAXY_APP will be removed from your phone._  

Has someone got this error message on the Android smartphone?

**Comment by tgardner** [comment]
Hi @Truckster78,
It sounds like you haven't installed a developer certificate from Samsung. Please read the documentation http://developer.samsung.com/z/develop/getting-certificates

Thanks,
Trent

**Comment by Truckster78** [comment]
Hi tgardner,
thanks for your answer. 

In fact, I managed the certificate from Samsung on the Tizen side. It works, but do I have to do the same thing on the Ionic/Cordova side? If yes, where do I managed that?

Do I need to declare, in the Ionic/Cordova project, the Tizen's app name that I communicate to?

**Comment by Truckster78** [comment]
Hello tgartner, 

you are right. The problem came from the Samsung's certificate. In fact, I have two computers, one with Tizen and one with Ionic. 
I thought (I am beginner) that I have to specify the certificate in my project, but not. 
I installed the Tizen certificate on the second machine; through Tizen application and "Certificate manager". And when Ionic/Cordova builds my project, it takes the certificate in the good folder (c:\users\USERNAME\samsung_certificates\...\CertificateName.p12) to add to the apk file.
Now, I do not have anymore the error message.

Thanks for your help.

**Comment by Truckster78** [comment]
Hello tgartner,

you are right. The problem came from the Samsung's certificate. In fact, I have two computers, one with Tizen and one with Ionic.
I thought (I am beginner) that I have to specify the certificate in my ionic's project, but not.
I installed the Tizen certificate on the second machine; through Tizen application and "Certificate manager". And when Ionic/Cordova builds my project, it takes the certificate in the good folder (c:\users\USERNAME\samsung_certificates...\CertificateName.p12) to add to the apk file.
Now, I do not have anymore this error message.

But I have got another, the Android one, that says the "App is stopped". This message pop up each time I request your plugin function, for example _Galaxygear.onConnect_... In Chrome-Inspect, tab _console_, I do not see any error message...
Any idea?!
